JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -73,7 +74,7 @@ ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
 
 		# check for errors
 		anyErrors <- .hasErrors(dataset = dataset, perform = perform,
-								type = c("infinity", "variance"))
+								type = c("infinity", "variance", "observations"), observations.amount = " <= 2")
```

Comment:
It's more intuitive (and consistent with other analyses) to use observations.amount = "< 3"